axiom/nr_log_event_private.h

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef NR_LOG_EVENT_PRIVATE_H
+#define NR_LOG_EVENT_PRIVATE_H
+
+#include "nr_log_event.h"
+#include "util_object.h"
+
+struct _nr_log_event_t {
```

Comment:
This is more nebulous and follows the logic of _nr_span_event_t. I believe it belongs in a separate "private" file because the members of this struct should never be accessed in any context other than calling the "set" methods. Having the "get" methods (for testing purposes) and members defined in a "private" file somewhat enforces this line of thinking.